From 8a6002f9d58aad14fa5c7e0923353165f8e6fe7a Mon Sep 17 00:00:00 2001
From: cchoux <cosmo.chou@quantatw.com>
Date: Sat, 3 Dec 2022 12:37:31 +0800
Subject: [PATCH] workaround for CX7 set MAC Affinity issue

Summary:
Set Host Number of MAC Affinity to 0 for workaround
---
 net/ncsi/ncsi-manage.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/ncsi/ncsi-manage.c b/net/ncsi/ncsi-manage.c
index 20acbc86f2b..a7bbe07aa7f 100644
--- a/net/ncsi/ncsi-manage.c
+++ b/net/ncsi/ncsi-manage.c
@@ -799,7 +799,7 @@ static int ncsi_oem_smaf_mlx(struct ncsi_cmd_arg *nca)
 	u.data_u32[0] = ntohl((__force __be32)NCSI_OEM_MFR_MLX_ID);
 	u.data_u8[5] = NCSI_OEM_MLX_CMD_SMAF;
 	u.data_u8[6] = NCSI_OEM_MLX_CMD_SMAF_PARAM;
-	u.data_u8[7] = 0xFF;
+	u.data_u8[7] = 0x00;
 	memcpy(&u.data_u8[MLX_SMAF_MAC_ADDR_OFFSET],
 	       nca->ndp->ndev.dev->dev_addr,	ETH_ALEN);
 	u.data_u8[MLX_SMAF_MED_SUPPORT_OFFSET] =
-- 
2.25.1

